ggshield/utils/files.py

```diff
@@ -13,11 +13,15 @@ def __init__(self, path: Path):
         self.path = path
 
 
-def is_filepath_excluded(
-    filepath: Union[str, Path], exclusion_regexes: Set[re.Pattern]
+def is_path_excluded(
+    path: Union[str, Path], exclusion_regexes: Set[re.Pattern]
 ) -> bool:
-    filepath = Path(filepath)
-    return any(r.search(str(PurePosixPath(filepath))) for r in exclusion_regexes)
+    path = Path(path)
+    if path.is_dir():
+        path_string = f"{PurePosixPath(path)}/"
```

Comment:
Following our discussions, can you add a comment here to explain why we want the trailing slash?